ocs_ci/helpers/sanity_helpers.py

```diff
@@ -68,68 +71,21 @@ def create_resources(self, pvc_factory, pod_factory, run_io=True):
                 pod.run_io("fs", "1G", runtime=30)
             for pod in self.pod_objs:
                 get_fio_rw_iops(pod)
-        self.create_obc()
-        self.verify_obc()
 
-    def create_obc(self):
-        """
-        OBC creation for RGW and Nooba
-
-        """
-        if config.ENV_DATA["platform"] in constants.ON_PREM_PLATFORMS:
-            obc_rgw = templating.load_yaml(constants.RGW_OBC_YAML)
-            obc_rgw_data_yaml = tempfile.NamedTemporaryFile(
-                mode="w+", prefix="obc_rgw_data", delete=False
-            )
-            templating.dump_data_to_temp_yaml(obc_rgw, obc_rgw_data_yaml.name)
-            logger.info("Creating OBC for rgw")
-            run_cmd(f"oc create -f {obc_rgw_data_yaml.name}", timeout=2400)
-            self.obc_rgw = obc_rgw["metadata"]["name"]
-
-        obc_nooba = templating.load_yaml(constants.MCG_OBC_YAML)
-        obc_mcg_data_yaml = tempfile.NamedTemporaryFile(
-            mode="w+", prefix="obc_mcg_data", delete=False
-        )
-        templating.dump_data_to_temp_yaml(obc_nooba, obc_mcg_data_yaml.name)
-        logger.info("create OBC for mcg")
-        run_cmd(f"oc create -f {obc_mcg_data_yaml.name}", timeout=2400)
-        self.obc_mcg = obc_nooba["metadata"]["name"]
-
-    def delete_obc(self):
-        """
-        Clenaup OBC resources created above
-
-        """
         if config.ENV_DATA["platform"] in constants.ON_PREM_PLATFORMS:
-            logger.info(f"Deleting rgw obc {self.obc_rgw}")
-            obcrgw = OCP(kind="ObjectBucketClaim", resource_name=f"{self.obc_rgw}")
-            run_cmd(f"oc delete obc/{self.obc_rgw}")
-            obcrgw.wait_for_delete(resource_name=f"{self.obc_rgw}", timeout=300)
-
-        logger.info(f"Deleting mcg obc {self.obc_mcg}")
-        obcmcg = OCP(kind="ObjectBucketClaim", resource_name=f"{self.obc_mcg}")
-        run_cmd(
-            f"oc delete obc/{self.obc_mcg} -n " f"{defaults.ROOK_CLUSTER_NAMESPACE}"
-        )
-        obcmcg.wait_for_delete(resource_name=f"{self.obc_mcg}", timeout=300)
+            self.obc_objs.append(rgw_bucket_factory(1, "rgw-oc")[0])
```

Comment:
Another nicety I thought of too late (but would still like to suggest) - it's possible to use `.extend` without the need to mention an index at all -
```suggestion
            self.obc_objs.extend(rgw_bucket_factory(1, "rgw-oc"))
```
Can also be applied to line 78 :)

Reply:
done